tock/hours/forms.py

```diff
@@ -101,7 +110,7 @@ def clean(self):
             # print(form)
             if form.cleaned_data:
                 # Easy way of telling if we have the right data
-                if form.cleaned_data.get('hours_spent') == None:
+                if not form.cleaned_data.get('hours_spent'):
```

Comment:
Should we be checking for `None` or for falsy values here? If `hours_spent` is allowed to be zero, we should stick with the `is None` check, else this is good.